.travis.yml

```diff
@@ -0,0 +1,5 @@
+sudo: required
+services:
+-   docker
+script:
+-   docker run -v $(pwd):/code:rw team5499/frc-testing-image sh -c "cd /code && ./test.sh"
```

Comment:
Why not just run `/code/gradlew -p /code test` instead of `sh -c ...`?

Reply:
good idea, I'll change it